CHANGELOG.md

```diff
@@ -4,6 +4,95 @@
 
 # JupyterLab Changelog
 
+## v4.4
+
+JupyterLab 4.4 includes a number of new features (described below), bug fixes, and enhancements.
+This release is compatible with extensions supporting JupyterLab 4.0.
+Extension authors are recommended to consult the [Extension Migration Guide](https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-4-3-to-4-4) which lists deprecations and changes to the public API.
+
+### Code console improvements
+
+The code console prompt can now be positioned on top, left, or right side of the console,
+in addition to the default bottom placement. The placement is persisted in the Prompt Cell
+Position setting.
+
+The console toolbar now includes buttons to:
+- execute code,
+- restart the kernel,
+- clear the cells,
+- switch the kernel.
+
+Additional settings for console behavior were added:
+- Clear Code Content on Execute (enabled by default) - when disabled,
+  the code submitted for execution remains in the code editor, allowing for further edits
+- Hide Code Input - when enabled shows only the output of the execution in the console
+- Clear Cells on Execute - when enabled shows only last cell
+
+Toggling all the three new settings transforms the console into an interactive editor
```

Comment:
Maybe we could add a screenshot to better what it looks like when all the options are enabled?

For example this one from the related PR? https://github.com/jupyterlab/jupyterlab/pull/13837

![image](https://github.com/user-attachments/assets/41e2a4f2-90d2-45bc-86ee-30244025be69)

Reply:
Although it had the dev-mode red border at the top that could be removed.